### DISCUSSION TITLE
2007/05/23/palette-api

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#163] [Announcements]
# 2007/05/23/palette-api

对函数和控件面板的调整，并不是只有手工调整着一个方法。LabVIEW 8.6 中提供了在程序中修改函数和控件面板的 API。通过这些 API VI，用户就可以在程序里读取或设置 .mnu 文件里的内容，从而通过编程来改变函数和控件面板。

https://blog.qizhen.xyz/2007/05/23/palette-api

**Comment by ruanqizhen** [comment]
之前博客下的留言：

Qizhen
2007年05月31日 08:30 编辑
LabVIEW 下一个版本啥时候发布，我不便说啊。公司有专门人负责向客户公布这类信息，其他人了解的不清楚，说错了会给公司造成麻烦的。

回复
玲
2007年05月24日 12:50 编辑
     大哥，看完之后好佩服。我是计算机应用与维护毕业的学生，现在做丰与专业不对口的工作，很喜欢你的BLOG。就好像善人萨姆一样地。把所知所遇都拿来给大家分享。你加我好不好。

回复
liang
2007年05月23日 22:46 编辑
请问LV是否要发行新的版本了，具体时间什么时候啊
